data/sidebars/setup.yml

```diff
@@ -17,3 +17,7 @@
   items:
     - title: Run a Keeper
       link: /setup/keeper/
+- group: Compute-to-Data Enviroment
+  items:
+    - title: Run a Compute-to-Data Enviroment
+      link: /setup/ctd/
```

Comment:
we are fine to spell it out, which is also better for SEO: `/setup/compute-to-data/`